launchflow/gcp/compute_engine_service.py

```diff
@@ -231,20 +239,8 @@ def outputs(self) -> DockerServiceOutputs:
         service_url = None
         dns_outputs = None
         if self._custom_domain is not None:
-            service_url = f"https://{self._custom_domain.domain}"
-            try:
-                custom_domain_outputs = self._custom_domain.outputs()
-            except exceptions.ResourceOutputsNotFound:
-                raise exceptions.ServiceOutputsNotFound(service_name=self.name)
-            dns_outputs = DNSOutputs(
-                domain=custom_domain_outputs.registered_domain,
-                dns_records=[
-                    DNSRecord(
-                        dns_record_type="A",
-                        dns_record_value=custom_domain_outputs.ip_address,
-                    )
-                ],
-            )
+            dns_outputs = self._custom_domain.dns_outputs()
```

Comment:
nit: maybe wrap this in a ResourceOutputsNotFound and raise ServiceOutputsNotFound to match the others

Reply:
oh nvm looks like it might happen in the base class